build/postgres/Dockerfile

```diff
@@ -22,15 +22,7 @@ RUN apt-get update && \
     && rm -rf /var/lib/apt/lists/*
 
 # hadolint ignore=DL3003
-RUN git clone https://github.com/metabrainz/postgresql-musicbrainz-unaccent.git /tmp/postgresql-musicbrainz-unaccent \
-    && git clone https://github.com/metabrainz/postgresql-musicbrainz-collate.git /tmp/postgresql-musicbrainz-collate \
-    && git clone https://github.com/omniti-labs/pg_amqp.git /tmp/pg_amqp \
-    && cd /tmp/postgresql-musicbrainz-unaccent \
-    && make \
-    && make install \
-    && cd /tmp/postgresql-musicbrainz-collate \
-    && make \
-    && make install \
+RUN git clone https://github.com/omniti-labs/pg_amqp.git /tmp/pg_amqp \
```

Comment:
Here should also probably be a commit or version pinned.

Reply:
Same as above.